acceptance/config/nodes/debian-7-passenger.yaml

```diff
@@ -0,0 +1,20 @@
+---
+HOSTS:
+  master:
+    roles:
+    - master
+    platform: debian-wheezy-x86_64
+    hypervisor: vcloud
+    template: debian-7-x86_64
+    passenger: true
```

Comment:
Trying to override is_puppetserver here doesn't appear to work, so I'm going to make these generic server configs and do all the passenger vs puppetserver selection in options files.